### ISSUE TITLE
io.js support?

### ISSUE THREAD

**Issue by marbemac** [thlorenz/proxyquire#51]
Hi, everything works OK in node 0.12, but in io.js 1.3 I get the error below. Looks like for some reason mdl is read-only? Does anybody have this working in io.js?

The error occurs here: https://github.com/thlorenz/proxyquire/blob/master/lib/proxyquire.js#L15

```
if (!mdl[key])  mdl[key] = original[key];
                               ^
TypeError: Cannot assign to read only property 'F_OK' of #<Object>
```

**Comment by thlorenz** [comment]
I've no idea why that happens.
Could you reference this is in an io.js issue please? It might be an io.js regression.

**Comment by bendrucker** [comment]
Fixed in 16f66f3f811cd615b8a1ff520a89e3fee6bad8db
